src/inmanta_ui/config.py

```diff
@@ -16,20 +16,104 @@
     Contact: code@inmanta.com
 """
 from inmanta.config import Option, is_bool, is_str
+from inmanta.server.config import dash_auth_url, dash_client_id, dash_enable, dash_path, dash_realm
 
-web_console_enabled = Option("web-console", "enabled", True, "Should the server should host the web-console or not", is_bool)
+web_console_enabled = Option(
```

Comment:
I like the banner above the dashboard-related configuration options. Let's also add one above the old and the new configuration options related to the web-console.